index.html

```diff
@@ -270,16 +270,6 @@ <h2>Modifications to existing procedures</h2>
         line, as specified in [[RFC5285]], section 5, with direction taken from "e"'s direction
         attribute."
       </p>
-      <p>
-        In the algorithm for generating subsequent offers in [[RTCWEB-JSEP]] section 5.2.2, replace "The
-        RTP header extensions MUST only include those that are present in the most recent answer"
-        with "For each RTP header extension listed in [[\HeaderExtensionsToOffer]] that is
-        also present in the most recent answer, and where direction is not "stopped", generate
-        an appropriate "a=extmap" line with "direction" set according to the rules of [[RFC5285]]
-        section 6, considering the direction in [[\HeaderExtensionsToOffer]] to indicate the
-        answerer's desired usage". This preserves the property that the set of extensions
```

Comment:
and that means the property is not preserved, so you have to delete "This preserves the property.... " too.

Reply:
Done.